core/winnt/src/TWinNTSystem.cxx

```diff
@@ -4579,7 +4582,7 @@ TTime TWinNTSystem::Now()
 
 void TWinNTSystem::Sleep(UInt_t milliSec)
 {
-   ::Sleep(milliSec);
+   std::this_thread::sleep_for(std::chrono::milliseconds(milliSec));
```

Comment:
Do we also need to do the same for `::Sleep` (and/or is it not in use and/or what is special about `sleep_for`)?

Reply:
> Do we also need to do the same for `::Sleep` (and/or is it not in use and/or what is special about `sleep_for`)?

It's a hack to make the `_Thrd_sleep_for@4` symbol visible. I'll check if others will be needed...